### ISSUE TITLE
Permission exception when closing indices while being admin

### ISSUE THREAD

**Issue by Soulou** [floragunncom/search-guard#715]
Elasticsearch version: 6.6.1

When trying to restore a snapshot, I'm closing all indices except searguard one (which is not in the snapshot either)

But the operation fails with a 403 error and a `no permissions for[] and User` error. However, the current user is admin and has all permissions.

Is there something I'm missing about these permissions?

Regards,

```
$ curl 'http://elasticsearch-host:port/*,-searchguard/_close' -H 'Content-Type: application/json' -d '' | json_pp
{
   "error" : {
      "type" : "security_exception",
      "reason" : "no permissions for [] and User [name=sg_anonymous, roles=[sg_anonymous_backendrole], requestedTenant=null]",
      "root_cause" : [
         {
            "type" : "security_exception",
            "reason" : "no permissions for [] and User [name=sg_anonymous, roles=[sg_anonymous_backendrole], requestedTenant=null]"
         }
      ]
   },
   "status" : 403
}
```

```
# curl 'http://elasticsearch-host:port/_searchguard/authinfo' | json_pp
{
   "principal" : null,
   "sg_roles" : [
      "sg_all_access"
   ],
   "remote_address" : "192.168.100.2:55856",
   "sso_logout_url" : null,
   "user_requested_tenant" : null,
   "peer_certificates" : "0",
   "user_name" : "sg_anonymous",
   "user" : "User [name=sg_anonymous, roles=[sg_anonymous_backendrole], requestedTenant=null]",
   "sg_tenants" : {
      "sg_anonymous" : true,
      "adm_tenant" : true,
      "test_tenant_ro" : true
   },
   "custom_attribute_names" : [],
   "backend_roles" : [
      "sg_anonymous_backendrole"
   ]
}
```

**Comment by floragunncom** [comment]
This is an knwon issue fixed with v25.3 (released last week for ES 6.7.x and 6.8.x)

Workaround is to list all indices you want to close like `http://elasticsearch-host:port/index*,myindex,anotherone,*logstash*/_close`

**Comment by Soulou** [comment]
Ok thanks for the info
